doc/source/whatsnew/v3.0.0.rst

```diff
@@ -63,6 +64,7 @@ Other enhancements
 - Support reading Stata 102-format (Stata 1) dta files (:issue:`58978`)
 - Support reading Stata 110-format (Stata 7) dta files (:issue:`47176`)
 
+
```

Comment:
```suggestion
```

Reply:
This is interesting, the "+" mark was actually added by pre-commit hooks.